### PR TITLE
docs: Update Readme

### PR DESCRIPTION
**Pull request by Varda003**
added new features to the about project in the file. changed the alignment.. Also added about how to contribute in this project. Last added support section as well

fix: #9

### CODE REVIEW

**Comment by Varda003** [comment]
HI @devXprite  Please merge my pull request

**Comment by Varda003** [comment]
Please @devXprite  merge my pull request

File: README.md [hunk 1]
```diff
@@ -62,6 +71,47 @@ To run GitGlance locally, follow these steps:
 ## Contributing
 Contributions are welcome! If you encounter any issues or have suggestions for improvements, please feel free to open an issue or submit a pull request.
 
+
+## How to Contribute
+We welcome contributions from the community to enhance GitGlance further. Here's how you can contribute:
+
+# 1. Fork the Repository
```
Comment (devXprite) [inline README.md:78]: I think these headings are too big. They would be better suited as bullet points instead.

Screenshot at 2024-05-12 15-43-04

File: README.md [hunk 2]
```diff
@@ -62,6 +71,47 @@ To run GitGlance locally, follow these steps:
...
+Fork the GitGlance repository by clicking the "Fork" button at the top right corner of this page. This will create a copy of the repository in your GitHub account.
+
+# 2. Clone the Repository
+Clone your forked repository to your local machine using GitHub Desktop or by running the following command in your terminal:
+
+```bash
+git clone https://github.com/your-username/gitglance.git
+
```
Comment (devXprite) [inline README.md:86]: I think there is something wrong. Please fix it.

Screenshot at 2024-05-12 15-46-03

**Review by devXprite — CHANGES_REQUESTED** [review]

**Comment by Varda003** [comment]
Okay working on it. Will change that

**Comment by Varda003** [comment]
Hello @devXprite  Check I have made all the changes as you wanted it looks much better  now please review it accept the pull request

**Review by devXprite — APPROVED** [review]

**Comment by devXprite** [comment]
> Hello @devXprite Check I have made all the changes as you wanted it looks much better now please review it accept the pull request

Thanks @Varda003. Now it's looking great 👍

**Pull request merged**